Assets/_GameJamCat/Scripts/Managers/UIManager.cs

```diff
@@ -8,30 +8,36 @@ namespace GameJamCat
     public class UIManager : MonoBehaviour
     {
         private readonly IStateManager _stateManager = StateManager.Instance;
-        
+
+        [SerializeField] private ActionBoxMenu _actionBoxMenu = null;
+        [SerializeField] private ActionBoxView _actionBoxView = null;
+
         [Title("Screen Transition")]
+
         [SerializeField] private ScreenTransitionViewBehaviour _transitionViewBehaviour = null;
         [SerializeField] private PregameDialogueBoxBehaviour _pregameDialogueBox = null;
-        
+
         [Title("Dossier")]
         [SerializeField] private DossierViewBehaviour _dossierView = null;
-        
+
         [Title("End Game")]
         [SerializeField] private EndGameMenu _endgameViewBehaviour = null;
-        
+
         [Title("HUD View")]
         [SerializeField] private TimerUI _timer = null;
         [SerializeField] private GameObject _crossHair = null;
         [SerializeField] private LivesViewBehaviour _livesView = null;
+
         [SerializeField] private GameObject _crossHairText = null;
-        
+        [SerializeField] private PlayerController _player = null; //I'm sorry for doing this I know its terrible
```

Comment:
you can just pass it in through worldmanager

Reply:
ah shit didn't realise world manager had the player, will do

Reply:
Fixed